Automattic/wp-calypso
# Composite checkout: prevent svgs from being focusable on ie11

#### Changes proposed in this Pull Request
This PR prevents the browser from making the svg graphics focusable while you're tabbing through the checkout on ie11.

**Before**
![image](https://user-images.githubusercontent.com/6981253/69254770-c2cc9f00-0b84-11ea-9494-a4a034dcf00f.png)

**After**
![image](https://user-images.githubusercontent.com/6981253/69254811-cfe98e00-0b84-11ea-8806-e30d79ada46c.png)


#### Testing instructions
If you have IE it would be great to test it there but also check other browsers too. 

* Get checkout up and running with these instructions: https://github.com/Automattic/wp-calypso/pull/37013
*  Tab through the interface
